components/playground/instance/tikv.go

```diff
@@ -65,7 +65,7 @@ func (inst *TiKVInstance) Start(ctx context.Context, version utils.Version) erro
 		fmt.Sprintf("--addr=%s", utils.JoinHostPort(inst.Host, inst.Port)),
 		fmt.Sprintf("--advertise-addr=%s", utils.JoinHostPort(AdvertiseHost(inst.Host), inst.Port)),
 		fmt.Sprintf("--status-addr=%s", utils.JoinHostPort(inst.Host, inst.StatusPort)),
-		fmt.Sprintf("--pd=%s", strings.Join(endpoints, ",")),
+		fmt.Sprintf("--pd-endpoints=%s", strings.Join(endpoints, ",")),
```

Comment:
Is it compatible with older version of tikv

Reply:
Yes. I've checked with at least TiKV v4.0 and it is fine.